src/data/finset/lattice.lean

```diff
@@ -186,22 +187,22 @@ begin
 end
 
 -- If we acquire sublattices
--- the hypotheses should be reformulated as `s : subsemilattice_sup_bot`
+-- the hypotheses should be reformulated as `s : subsemilattice_sup` with `order_bot`
```

Comment:
I don't think this change is accurate, we can't use mixins for subobjects.
It's not enough for the subobject to be over a type with a bot, it needs to also contain that bot.

Reply:
```suggestion
-- the hypotheses should be reformulated as `s : subsemilattice_sup_bot`
```